src/Bundle/Maker/MakeGrid.php

```diff
@@ -0,0 +1,285 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Sylius\Bundle\GridBundle\Maker;
+
+use Sylius\Component\Grid\Maker\Filter\Helper\GridHelperInterface;
+use Sylius\Component\Grid\Maker\Filter\ResourceHelperInterface;
+use Symfony\Bundle\MakerBundle\ConsoleStyle;
+use Symfony\Bundle\MakerBundle\DependencyBuilder;
+use Symfony\Bundle\MakerBundle\Generator;
+use Symfony\Bundle\MakerBundle\InputConfiguration;
+use Symfony\Bundle\MakerBundle\Maker\AbstractMaker;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Yaml\Yaml;
+use Webmozart\Assert\Assert;
+
+final class MakeGrid extends AbstractMaker
+{
+    public static $defaultActionTypes = [
```

Comment:
```suggestion
    private static $defaultActionTypes = [
```

Also, won't it be better to make this variable a constant?  In case we would like to change it, it should be injected.

Reply:
public/protected/private const?

Reply:
We should be as strict as possible. We will always be able to expose const, but we won't be able to reduce the scope. `private const`